### DISCUSSION TITLE
キーボール44で特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by you-rin-chi** [Yowkees/keyball#250] [General]
先日初めての自作キーボードとしてキーボール44を購入し、ビルドガイドやYouTubeを参考に組み上げました。
いくつか反応しないキーがあったため、過去の質問やチェッカーを使った抵抗チェックを行ってかなり改善できました。
しかし、添付写真の赤枠で囲った一箇所がどうしても反応しませんでした。
お手数をおかけしますが対策をご教授いただけると幸いです。
ダイオードの方向チェックはチェッカーで実施しています。
足りない情報などあればすぐに対応しますのでご指摘お願いします。
![IMG_2890](https://github.com/Yowkees/keyball/assets/136183150/302d1967-637d-48f3-90ab-2b4b364032e9)

**Comment by Yowkees** [comment]
![KB44_Keyfix](https://github.com/Yowkees/keyball/assets/84209772/89cb7ae9-47e9-410c-853f-9da4e3e9450f)
ご連絡ありがとうございます。
キーが１つ効かないということで了解しました。
丸で囲んだダイオードがテンプラしていないかご確認ください。
確認方法は、ダイオードの足をはんだごてで加熱すると、片方の端子が浮いていれば部品が脱落するので分かります。
以上、お手数ですがご対応よろしくお願いいたします。
